Source/Particles/Collision/BackgroundMCCCollision.H

```diff
@@ -0,0 +1,53 @@
+/* Copyright 2021 Roelof Groenewald
+ *
+ * This file is part of WarpX.
+ *
+ * License: ????
+ */
+#ifndef WARPX_PARTICLES_COLLISION_BACKGROUNDMCCCOLLISION_H_
+#define WARPX_PARTICLES_COLLISION_BACKGROUNDMCCCOLLISION_H_
+
+#include "Particles/MultiParticleContainer.H"
+#include "CollisionBase.H"
+#include <AMReX_ParmParse.H>
+
+class BackgroundMCCCollision
```

Comment:
Does the existing code put functionality like this in some sort of `namespace`?

Reply:
The most similar existing code is in `Source/Particles/Collision/PairWiseCoulombCollision.H` and that doesn't use a specific `namespace`.